treeherder/model/fixtures/repository.json

```diff
@@ -1145,7 +1145,7 @@
         "name": "go-bouncer",
         "url": "https://github.com/mozilla-services/go-bouncer",
         "branch": "master",
-        "active_status": "active",
+        "active_status": "onhold",
```

Comment:
No results shown, so... more than a year? What's the push expiration age? The github page has activity as recently as yesterday, so I don't know what's going on here...
https://treeherder.mozilla.org/#/jobs?repo=go-bouncer